slices/ncurses-base.yaml

```diff
@@ -0,0 +1,88 @@
+package: ncurses-base
+
+slices:
+  # the "data" slice contains all the /lib/terminfo files
+  data:
```

Comment:
are they all binary data files? (making sure `data` is the right slice name for it)

Reply:
Yeah, they are all compiled terminfo entry. Contains binary data inside them. For example:

```sh
$ file /lib/terminfo/x/xterm
/lib/terminfo/x/xterm: Compiled terminfo entry "xterm"
```